globus_automate_client/flows_client.py

```diff
@@ -160,7 +160,7 @@ def deploy_flow(
         available for execution on the Globus Automate Flows Service.
 
         :param flow_definition: A mapping corresponding to a Globus Flows
-            defintion.
+            definition.
```

Comment:
I can't explain how many times I've spelled this word in the past two weeks and how many times I've missed the second "i"